XenAdmin/TabPages/PerformancePage.cs

```diff
@@ -360,9 +360,10 @@ private void EditGraph()
 
         private void DeleteGraph()
         {
+            var formattableName = GraphList.SelectedGraph.DisplayName.Replace("&", "&&");

```

Comment:
You can use the existing string extension method EscapeAmpersands, see file XenModel/StringExtensions